server/routes/auth.routes/auth.router.js

```diff
@@ -0,0 +1,64 @@
+import express from 'express';
+import jwt from 'jwt-simple';
+import passport from 'passport';
+import crypto from 'crypto';
+import User from '../../db/Mongo/Schemas/UserSchema';
+import sendEmail from '../../core/nodemailer';
+const router = express.Router();
+
+const tokenForUser = user =>{
+    const timestamp = new Date().getTime();
+    return jwt.encode({ sub: user.id, iat: timestamp }, process.env.JWT);
+}
+
+router.post('/join', (req, res, next) => {
+    const email = req.body.email;
+    const password = req.body.password;
+    if (!email || !passport) {
+        return res.status(422).send({ err: 'You must provide email and password' });
```

Comment:
let's store error codes as a dictionary of values with meaningful names. Try to find out if there is already a package for this in the Internet